release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2891,6 +2891,47 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.40
+[id="ocp-4-17-40_{context}"]
+=== RHBA-2025:16133 - {product-title} {product-version}.40 bug fix update and security
+
+Issued: 24 September 2025
+
+{product-title} release {product-version}.40 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2025:16133[RHBA-2025:16133] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:16131[RHBA-2025:16131] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.40 --pullspecs
+----
+
+[id="ocp-4-17-40-enhancements_{context}"]
+==== Enhancement
+
+* With this update, the `cluster-etcd-operator` Operator now implements a multi-stage notification system for the `etcdDatabaseQuotaLowSpace` alert to proactively manage etcd storage quotas. This enhancement is designed to prevent API server instability by providing earlier warnings of low database space. As etcd disk space usage reaches 65%, 75% and 85%, administrators now receive alerts with a severity level of info, warning, or critical. (link:https://issues.redhat.com/browse/OCPBUGS-61337[OCPBUGS-61337])
```

Comment:
I'm not really sure if `cluster-etcd-operator` Operator is correct. I only see a few mentionings of `cluster-etcd-operator` in our docs. I do see several entries for "etcd Operator." You might want to confirm wit someone (maybe an etcd writer?) that it is, in fact, `cluster-etcd-operator` Operator.